content/application/guide.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Installing applications on Urbit"
++++
+
+## 1. Get on the network
+
+First, you’ll need to get on Urbit. You can quickly check out the network by [downloading the Port application](https://github.com/urbit/port) and booting up a comet–a temporary Urbit ID.
+
+For long-term use, you’ll want to get a permanent Urbit ID and consider a running your urbit in the cloud or on a hosted service.
+
+Read our [Getting Started guide](https://urbit.org/install) for details.
+
+## 2. Finding applications
+
+Urbit is still in its early stages, so you’ll need to join groups and browse around the network to discover new applications as they are being brought up.
+
+However, we can get started a simple app from Urbit developer [`~paldev`](/id/~paldev).
+
+## 3. Search for a Developer
```

Comment:
You're not really "Search"ing for a developer; rather, you're entering a known one. 

You can also enter an entire app shortcode into this search bar, much as you can with groups, e.g. ~paldev/pals can be pasted directly from one of these pages.

Reply:
That actually doesn't work right now — I put in a fix so that it does at https://github.com/urbit/urbit/pull/5738 though.